app/views/reports/user_last_sign_in_at_report.html.erb

```diff
@@ -0,0 +1,49 @@
+<% @colors = ['#F7464A', '#46BFBD', '#FFC870', '#949FB1'] %>
+<div class="row">
+  <div class="col-md-3">
+    <fieldset>
+      <legend>Last login</legend>
+      <ul style="padding-left: 0px; list-style-type: none">
+        <% count = 0 %>
```

Comment:
@SeanLF you might want to explore @counts.each_with_index do |item, index| - but to be honest it might be just as much code as what you have. each_with_index for a hash spits out a hash item with an index. I'd say the only advantage is you don't need to create the temporary count variable. But as with everything I'm saying, just something to explore:)